testsuite/podman_runner/12_run_proxy.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+set -euxo pipefail
+
+create_proxy_configuration() {
+
+  PROXY_UTILS=$HOME/proxy
```

Comment:
IMHO, the word PROXY_UTILS could bring some misunderstandings, as we already have the word tools for other official components, mgr-tools (aka uyuni-tools) and also for client-tools.

Reply:
Ok, changed to `PROXY_HOME_DIRECTORY`